sale_order_qty_change_no_recompute/models/sale_order.py

```diff
@@ -0,0 +1,21 @@
+# Copyright 2021 Tecnativa - Víctor Martínez
+# License AGPL-3.0 or later (https://www.gnu.org/licenses/agpl).
+
+from odoo import api, models
+
+
+class SaleOrderLine(models.Model):
+    _inherit = "sale.order.line"
+
+    @api.onchange(
+        "product_id", "price_unit", "product_uom", "product_uom_qty", "tax_id"
+    )
+    def _onchange_discount(self):
```

Comment:
Are you sure this is the proper method to override?

Reply:
Now addon override some methods: `product_uom_change` + `_onchange_discount` (it's necessary to prevent price recalculation)